todo_app/data/trello_items.py

```diff
@@ -1,102 +1,107 @@
 from todo_app.data.todo_items import Item
-import requests, os
+from datetime import date, datetime
+import os
+import pymongo
+from bson.objectid import ObjectId
+
+def get_connection():
+    client = pymongo.MongoClient(os.getenv("COSMOS_CONNECTION_STRING"))
+    endpoint = client
+    return endpoint
```

Comment:
Is this variable rename actually useful? If we think endpoint is a better name (and I wouldn't necessarily agree) then we should just name it that the first time surely - e.g.:
```python
endpoint = pymongo.MongoClient(os.getenv("COSMOS_CONNECTION_STRING"))
```